runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -58,6 +59,10 @@
 
     @Setter
     @Getter
+    Class defaultObjectsType = SMRMap.class;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SMRMap"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)